src/main/java/apka/config.json

```diff
@@ -0,0 +1 @@
+{"gpsPort":1911,"gpsIP":"0.0.0.0","gprsIP":"0.0.0.1","port":7779,"ip":"0.0.0.1","gprsPort":9111}
```

Comment:
No to przesłany JSON zapisuje się do hashmapy, więc jak wyślesz do mnie np. {"gps":[{"ip":"0.0.0.1","port":6699}]}, to pod key = gps, będę miał arraylist jego parametrów, więc chyba będzie ok, czy jeszcze inaczej chcesz?